app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -43,6 +56,13 @@ def validate_import_vm
     Gem::Version.new(api_version) >= Gem::Version.new('4')
   end
 
+  def submit_configure_imported_vm_networks(userid, vm_id)
```

Comment:
@matobet You are correct.  You are passing the `userid` column from the `User` class.  (Not to be confused with the `id` column from the user class.)  😕 

See usage here: https://github.com/ManageIQ/manageiq-content/pull/123/files#diff-20380aa1ea403e71a951c411b93702f6R22

As far as naming goes, can we drop the `submit_` prefix?